go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/alphagov/paas-rubbernecker
 
-go 1.14
+go 1.16.4
```

Comment:
`go test` does not like defining the patch: 
```
go: errors parsing go.mod:
/home/sebastianschmieschek/paas/paas-rubbernecker/go.mod:3: invalid go version '1.16.4': must match format 1.23
```
using `go 1.16` all tests pass.

Reply:
updated